ote/src/cljs/ote/views/transport_service_common.cljs

```diff
@@ -419,9 +419,15 @@
 
         (if published?
           ;; True
-          [buttons/save {:on-click #(e! (ts/->SaveTransportService schemas true))
-                         :disabled (not (form/can-save? data))}
-           (tr [:buttons :save-updated])]
+          [:span
+           [buttons/save {:on-click #(e! (ts/->SaveTransportService schemas true))
+                          :disabled (not (form/can-save? data))}
+            (tr [:buttons :save-updated])]
+           [buttons/save {:disabled name-missing?
```

Comment:
After pressing :back-to-draft once, save should be enabled and back-to-draft disabled.
Will current :disabled condition leave it enabled?

Reply:
Back to Draft - button is only shown when transport service is published.
It is inside 
(if published? )